web/frontend/src/components/gamepad/index.svelte

```diff
@@ -252,7 +252,7 @@ onMount(() => {
   tick();
 });
 
-useDisconnect(() => clearTimeout(handle));
+onDestroy(() => clearTimeout(handle));
```

Comment:
Gamepad I omitted entirely because it's pretty broken / will be rewritten. This won't cause any regressions.